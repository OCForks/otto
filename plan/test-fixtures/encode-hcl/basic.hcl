plan {
    inputs {
    foo = "bar"
    bar = "baz"
    }

    task "test-1" {
        description = "desc foo"
    }

    task "test-1" {
        description = "desc foo"
        foo = "${result.Result}"
        bar = "baz"
        quux = "quux"
    }

    task "store" {
        foo = "${result.Result}"
    }

    task "test-1" {
        description = "desc foo"
        foo = "${foo}"
    }

    task "delete" {
        key = "foo"
    }
}
