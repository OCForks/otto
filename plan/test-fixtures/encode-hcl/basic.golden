plan {
  inputs {
    bar = "baz"
    foo = "bar"
  }

  task "test-1" {
    description = "desc foo"
  }

  task "test-1" {
    description = "desc foo"

    bar  = "baz"
    foo  = "${result.Result}"
    quux = "quux"
  }

  task "store" {
    foo = "${result.Result}"
  }

  task "test-1" {
    description = "desc foo"

    foo = "${foo}"
  }

  task "delete" {
    key = "foo"
  }
}